tests/cpu/ops/fixedfunction/FixedFunctionOpData_tests.cpp

```diff
@@ -66,6 +66,104 @@ OCIO_ADD_TEST(FixedFunctionOpData, aces_dark_to_dim10_style)
                           "The style 'ACES_DarkToDim10 (Forward)' must have zero parameters but 1 found.");
 }
 
+OCIO_ADD_TEST(FixedFunctionOpData, aces_gamut_map_13_style)
+{
+    OCIO::FixedFunctionOpData::Params params = { 1.147, 1.264, 1.312, 0.815, 0.803, 0.880, 1.2 };
+    OCIO::FixedFunctionOpData func(OCIO::FixedFunctionOpData::ACES_GAMUTMAP_13_FWD, params);
+    OCIO_CHECK_NO_THROW(func.validate());
+    std::string cacheID;
+    OCIO_CHECK_NO_THROW(cacheID = func.getCacheID());
+    OCIO_CHECK_ASSERT(func.getParams() == params);
+
+    OCIO::FixedFunctionOpDataRcPtr inv = func.inverse();
+    OCIO_CHECK_EQUAL(inv->getParams()[0], func.getParams()[0]);
+    OCIO_CHECK_EQUAL(inv->getStyle(), OCIO::FixedFunctionOpData::ACES_GAMUTMAP_13_INV);
+    std::string cacheIDUpdated;
+    OCIO_CHECK_NO_THROW(cacheIDUpdated = inv->getCacheID());
+    OCIO_CHECK_ASSERT(cacheID != cacheIDUpdated);
+
+    OCIO_CHECK_ASSERT(func == func);
```

Comment:
Intended?

Reply:
Not sure about it, it's from `rec2100_surround_style` test below, probably make not much sense to test that for two different style.

Reply:
It's fine to leave it.  I suppose it's worth verifying that the op== method handles the extra parameters correctly.